lib/Backend/GlobOpt.cpp

```diff
@@ -3674,6 +3674,14 @@ GlobOpt::CopyProp(IR::Opnd *opnd, IR::Instr *instr, Value *val, IR::IndirOpnd *p
 
     ValueInfo *valueInfo = val->GetValueInfo();
 
+
+    if (instr->m_opcode == Js::OpCode::ArgOut_A_Inline && valueInfo->GetSymStore() &&
+        valueInfo->GetSymStore()->m_id == 0)
+    {
+        // We don't want to copy-prop s0 (return symbol) into inlinee code
```

Comment:
It should be fine to do the copy-prop in non EH cases, no?

Reply:
The issue comes up because I added s0 definition at the top of the function for try - finally early exit transformations. The IR wouldn't like this without it.